### PR TITLE
Add gemini-2.0-flash-exp

### PR DESCRIPTION
**Pull request by ccqpein**
I cant find the `:path` in google's doc (they don't give the rest call example), I just make my best guess, looks like I am lucky. 

*I just tested on my machine*

### CODE REVIEW

File: chatgpt-shell-google.el [hunk 1]
```diff
@@ -94,7 +94,12 @@ VALIDATE-COMMAND handler."
                                          :short-version "1.5-flash"
                                          :path "/v1beta/models/gemini-1.5-flash"
                                          :token-width 4
-                                         :context-window 1048576)))
+                                         :context-window 1048576)
+        (chatgpt-shell-google-make-model :version "gemini-2.0-flash-exp"
+                                         :short-version "2.0-flash-exp"
+                                         :path "/v1beta/models/gemini-2.0-flash-exp"
+                                         :token-width 4
+                                         :context-window 2097152)))
```
Comment (mpereira) [inline chatgpt-shell-google.el:102]: The context window of `gemini-2.0-flash-exp` is `1048576`?

CleanShot 2024-12-20 at 19 12 44@2x

Comment (ccqpein) [reply chatgpt-shell-google.el]: Good catch, I just changed it.

Comment (mpereira) [reply chatgpt-shell-google.el]: Thanks!

**Review by xenodium — APPROVED** [review]

**Comment by xenodium** [comment]
Thank you both! Merged.

**Pull request merged**